PopPUNK/refine.py

```diff
@@ -180,24 +167,31 @@ def refineFit(distMat, sample_names, start_s, mean0, mean1,
             gradient = optimal_x / optimal_y # of 1D search
             delta = x_max[1] - x_max[0]
             bounds = [-delta, delta]
-            start_point = (optimal_x, 0)
             mean1 = (optimal_x + delta, delta * gradient)
 
     else:
+        # Set the range of points to search
+        search_length = max_move + ((mean1[0] - mean0[0])**2 + (mean1[1] - mean0[1])**2)**0.5
         global_grid_resolution = 40 # Seems to work
```

Comment:
Would it be helpful to print the search range here, if not being returned elsewhere? Just so the users are aware of the change in input, and for reproducibility if needed.

Reply:
Yep, very good idea